glide.yaml

```diff
@@ -19,14 +19,25 @@ import:
   - pkg/api
   - pkg/api/install
   - pkg/api/v1
+  - pkg/apis/batch/v1
   - pkg/apis/extensions/install
   - pkg/apis/extensions/v1beta1
-  - kubernetes/typed/admissionregistration/v1alpha1
 - package: k8s.io/apimachinery
+  version: 1fd2e63a9a370677308a42f24fd40c86438afddf
   subpackages:
   - pkg/api/resource
   - pkg/apis/meta/v1
   - pkg/runtime
   - pkg/util/intstr
 - package: github.com/sirupsen/logrus
   version: v1.0.3
+- package: github.com/fsouza/go-dockerclient
+  version: d6c9513c3abe61f13e41c22bc013346eb96e1fe5
+- package: github.com/novln/docker-parser
+  version: 6030251119d652af8ead44ac7907444227b64d56
+# pinning 'golang.org/x/sys' otherwise docker dependencies complain to compiling
+- package: golang.org/x/sys
+  version: 7ddbeae9ae08c6a06a59597f0c9edbc5ff2444ce
```

Comment:
can we use this in `8d1157a435470616f975ff9bb013bea8d0962067` version?

The same version is used by OpenShift -  this will make vendoring easier

Reply:
done!